content/en/metrics/guide/metrics-without-limits-getting-started.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Getting Started with Metrics without Limits™
+kind: guide
+is_beta: true
+---
+
+<div class="alert alert-warning">This functionality is in private beta.</div>
+
+## Overview
+
+This guide covers how to get started with Metrics without Limits™ and quickly gain the most value from the feature. 
+
+Review the [Metrics without Limits™ FAQ][1].
+
+
+1. Begin by configuring your Top 20 metrics on the Usage page.
+   You can [configure metrics][2] both in the Metrics Summary UI and with the API.
+
+   **Note:** If you're using the Create Tag Configuration API, use the [tag configuration cardinality estimator API][3] first to validate the potential impact of your tag configurations prior to using the API above to create tag configurations. 
+
+   If the UI or the estimator API returns a resulting number of indexed that is [larger than ingested](#why-is-my-indexed-cm-volume-greater-than-ingested-cm-volume), do not save your tag configuration. 
+
+2. Use Bulk Metric Configuration to quickly configure tags on multiple metrics.
```

Comment:
Is this a variation of how to configure tags using the UI? If so, it should be a substep of that step, not a next step.

Reply:
Could be a substep but is only applicable as a substep if the # of indexed custom metrics is > than the metric's original size of ingested custom metrics.